app/comments/service.py

```diff
@@ -94,34 +96,39 @@ async def get_child_comments(
 
 
 async def create_comment(
-    session: AsyncSession,
-    user_id: int,
-    post_id: int,
-    comment_data: CommentsCreateRequest,
-):
-    ...
-    # depth = 0
-    # if comment_data.parent_id:
-    #     stmt = select(Comment).where(Comment.id == comment_data.parent_id)
-    #     result = await session.execute(stmt)
-    #     parent_comment = result.scalar_one_or_none()
-
-    #     if not parent_comment:
-    #         raise HTTPException(status_code=404, detail="부모 댓글을 찾을 수 없습니다.")
-
-    #     depth = parent_comment.depth + 1
-
-    # db_comment = Comment(
-    #     author_id=user_id,
-    #     post_id=post_id,
-    #     parent_id=comment_data.parent_id,
-    #     detail=comment_data.detail,
-    #     depth=depth,
-    # )
+    session: AsyncSession, author: User, comment: CommentCreate, post_id: int
```

Comment:
이런식으로 키워드 각각이 별개의 의미를 가지는 경우에는, 
- `create_comment(a, b, c)` (positional argument)
- `create_comment(session=a, author=b, comment=c)` (keyword argument)
이렇게 두 가지 다른 방식으로 사용할 수 있게 되는데요, 

대부분 전자의 경우에는 의도가 명확하게 드러나지 않아 가독성이 떨어지기 때문에 후자의 방식으로 사용하는 것이 더 좋습니다.
이를 정의부에서 강제할 수 있는데요, 이렇게 argument로 *을 넣어주는 경우 그 이후의 argument들은 kwargs로 강제되게 되어서, 넣어주면 좋습니다

```suggestion
    *, session: AsyncSession, author: User, comment: CommentCreate, post_id: int
```

Reply:
*을 사용하는 이유는 kwargs 임을 명시하는 기능만 있는걸까요?